R/endpoints.R

```diff
@@ -122,6 +123,36 @@ endpoint_model_result <- function(queue) {
   }
 }
 
+endpoint_model_debug <- function(queue) {
+  function(req, res, id) {
+    response <- with_success(queue$queue$task_data(id))
+    if (!response$success) {
+      response$errors <- hintr_errors(list(
+        "INVALID_TASK" = response$message))
+      res$status <- 400
+      return(hintr_response(response, "ModelDebugResponse"))
```

Comment:
Is this schema in this PR? I'm surprised this doesn't fail some tests.

Reply:
Oh yeah, this just does not get triggered I think because of the error handler. I was a bit confused about other cases here (and we don't really return json)